### PR TITLE
update 1.X -> 4.X, demo the styles

### PR DESCRIPTION
**Pull request by jakswa**
- big update here, more work to do
- demoing the [`styled(Component)`](https://material-ui.com/styles/basics/) pattern here for the recommended styling pathway moving forward (if we like it... alternate path is doing this update without any of the style-related motivations)
- put this on https://beta.marta.io so you can look at the current ugliness:

image

### CODE REVIEW

**Comment by jakswa** [comment]
still got some oddities on the station view but this is closer

File: src/StationListItem/StationListItem.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ class StationListItem {
     const arrivals = this.arrivalsByDirection(arrivalData, stationName);
 
     return (
-      <ListItem divider key={stationName} component={Link} to={link}>
+      <ListItem divider button key={stationName} component={Link} to={link}>
         <ListItemText className="station-list-title" primary={stationName.replace(/ station$/i, '')} />
         <div className="ContainerStationPills">{this.renderPills(arrivals)}</div>
```
Comment (autumncalhoun) [inline src/StationListItem/StationListItem.js]: I wrote this line previously, all old-skool-like, but I should have done
```
<Box className="ContainerStationPills">{this.renderPills(arrivals)}</Box>
```

File: src/StationPills/StationPills.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
+import { styled } from '@material-ui/core/styles';
 import Chip from '@material-ui/core/Chip';
 import Avatar from '@material-ui/core/Avatar';
 import Icon from '@material-ui/core/Icon';
 
 const BLANK_CHIP = <Chip className="BlankChip" style={{opacity: 0.5}} label="NO DATA" />;
 
+const MyChip = styled(Chip)({
```
Comment (autumncalhoun) [inline src/StationPills/StationPills.js]: 🎉  could we name this `ChipWithCountdown` or `TimerChip` or something?

File: public/index.html [hunk 3]
```diff
@@ -64,7 +64,6 @@
       .Pill {
         margin-left: 5px;
         font-weight: 500;
-        font-family: monospace !important;
```
Comment (autumncalhoun) [inline public/index.html:67]: yay!

**Comment by jakswa** [comment]
merged in @autumncalhoun's cleanup here, and I think it looks good enough to merge. I almost did merge it before noticing the direction font size seems to have taken a hit, and become smaller:

(notice the difference in the N/S/E/W lettering)
image

**Comment by jakswa** [comment]
Ok I think I have this close, I'm going to get it up on beta.marta.io in a bit:

image

File: public/index.html [hunk 4]
```diff
@@ -64,45 +70,48 @@
...
+        top: -3px;
+      }
+
+      .ContainerStationPills :not(.BlankChip) .MuiChip-label {
+        font-family: monospace;
+        font-weight: 700;
+        font-size: 1.4rem;
+      }
+
+      .MuiChip-root .Badge.MuiChip-avatar {
```
Comment (autumncalhoun) [inline public/index.html:92]: Sorry about that, my PR didn't change the margin or default height/width of the circle but I thought I fixed the small font. Anyway, it looks good now on my local.

Comment (jakswa) [reply public/index.html:92]: fwiw I don't think any of that was your doing. I think it was the version upgrade's fault.

**Pull request merged**